packages/comet-uswds/src/components/alert/alert.stories.tsx

```diff
@@ -26,3 +26,18 @@ Default.args = {
   noIcon: false,
   heading: '',
 };
+
+const BodyTemplate: StoryFn<typeof Alert> = (args: AlertProps) => (
+  <Alert {...args}>This is the alert body</Alert>
+);
+
+export const WithBody = BodyTemplate.bind({});
+WithBody.args = {
+  id: 'alert-1',
```

Comment:
The id's are the same, for **const Default** and  **const WithBody** not sure if these would ever be used at the same time.